Dockerfile

```diff
@@ -1,8 +1,23 @@
-FROM debian:jessie
+FROM golang:1.10.1-alpine3.7 as builder
 
-COPY ./bin/ocagent_linux ocagent
+RUN apk update && apk upgrade && \
+    apk add --no-cache bash git openssh
 
-# Expose the OpenCensus interceptor port
+ENV GOPKG github.com/census-instrumentation/opencensus-service
+
+COPY . /go/src/$GOPKG/
```

Comment:
Interesting, so now the build will be done by the Docker image itself right?

Reply:
Yes, exactly.